axios/apiClient.ts

```diff
@@ -0,0 +1,33 @@
+import React from "react";
+import axios from "axios";
+
+const client = axios.create({
+  headers: {
+    "Content-Type": "application/json",
+    Accept: "application/json",
+    ...(process.env.NODE_ENV === "development" && {
+      "Access-Control-Allow-Origin": "*",
+    }),
+    // ...(token && { "auth": "TOKEN"})
+  },
+  baseURL: "http://localhost:8000/api/",
+  withCredentials: true,
+});
+
+client.interceptors.request.use(
```

Comment:
What's this doing ?

Reply:
this is so we can add the auth headers before each request, I dont actually think we need it anymore.